dev/generate_rule_metadata.py

```diff
@@ -113,10 +113,14 @@ def read_existing_default_value(rule_number, benchmark_id):
     rule_dir = os.path.join(common.rules_dir, f"{benchmark_id}/rules", f"cis_{rule_number.replace('.', '_')}")
     try:
         with open(os.path.join(rule_dir, "data.yaml"), "r") as f:
-            default_value = yml.load(f)["metadata"]["default_value"]
-            return default_value
+            data = yml.load(f)
+            default_value = data["metadata"]["default_value"]
+            if default_value is None or default_value == "":
```

Comment:
a default value of an empty string would work in the sense that kibana won't [show](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/overview_tab.tsx#L116-L125) it.

technically the interface is `null | string` 

so i'd say we should use `null` as a fallback default value for every rule without one, instead of adding manually.  an empty string would work just the same but less explicit IMO